pyshield/_config.py

```diff
@@ -8,12 +8,20 @@
 from dacite import Config, from_dict
 
 from ndsl import MetaEnumStr
+from ndsl.dsl.gt4py_utils import tracer_variables
+from ndsl.dsl.typing import Float, set_4d_field_size
 from ndsl.utils import f90nml_as_dict
 
 
+# TODO: This is a hack and needs to be set at runtime by n_tracers
+# A full Tracer implementation should resolve this
+FloatFieldTracer = set_4d_field_size(9, Float)
+
 DEFAULT_INT = 0
 DEFAULT_BOOL = False
+DEFAULT_FLOAT = 0.0
```

Comment:
```suggestion
DEFAULT_FLOAT = Float(0.0)
```

That would start hardening code against mixed precision nightmare down the road